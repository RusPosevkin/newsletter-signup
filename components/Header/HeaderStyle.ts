import styled from "styled-components";

export const HeaderContainer = styled.div`
  display: flex;
  align-items: center;
  justify-content: center;
  width: 100%;
  min-height: 100px;
  border-bottom: 4px dashed rgb(0, 150, 136);
  color: rgb(0, 150, 136);
  font-size: 3rem;
  font-weight: bold;
`;
