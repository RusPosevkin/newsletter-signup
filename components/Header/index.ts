import HeaderComponent from './HeaderStructure';

export default HeaderComponent;
