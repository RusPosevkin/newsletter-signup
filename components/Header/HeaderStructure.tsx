import React from 'react'
import Link from 'next/link';
import { HeaderContainer } from './HeaderStyle';

const HeaderStructure = () => (
  <HeaderContainer>
    <Link href="/">Shop</Link>
  </HeaderContainer>
)
export default HeaderStructure;
