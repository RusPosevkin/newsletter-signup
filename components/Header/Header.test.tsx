import { render, screen } from '@testing-library/react'
import '@testing-library/jest-dom'
import Header from '.'

test('it should match the snapshot and render', async () => {
  const { container } = render(<Header />);
  expect(container).toMatchSnapshot();
  expect(container).toBeInTheDocument();
});
