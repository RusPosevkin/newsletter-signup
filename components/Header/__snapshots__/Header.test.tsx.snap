// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it should match the snapshot and render 1`] = `
<div>
  <div
    class="sc-e64e39fe-0 gzxShV"
  >
    <a
      href="/"
    >
      Shop
    </a>
  </div>
</div>
`;
